src/core/server/docs/kib_core_logging.mdx

```diff
@@ -323,6 +323,83 @@ The maximum number of files to keep. Once this number is reached, oldest files w
 
 The default value is `7`
 
+**deprecated**: Please use a retention policy instead.
+
+#### Retention Policies
+
+The retention policy defines when the rolled files must be deleted.
+
+##### maxFiles
```

Comment:
The added doc section.

I don't think we have any other public-facing documentation about logging, do we?